From 93deb778917cc761bd2b9384f813ed278fd98ba5 Mon Sep 17 00:00:00 2001
From: Adam Mercer <adam.mercer@ligo.org>
Date: Thu, 14 May 2020 15:14:43 -0700
Subject: [PATCH 1/2] lalapps: remove check for obsolete framel

we've required at least version 8.0 of the frame library for some time
---
 lalapps/src/frametools/animate.c | 4 ----
 lalapps/src/frametools/calfacs.c | 4 ----
 2 files changed, 8 deletions(-)

diff --git a/lalapps/src/frametools/animate.c b/lalapps/src/frametools/animate.c
index 3a282f08ef..80c2a2a6c1 100644
--- a/lalapps/src/frametools/animate.c
+++ b/lalapps/src/frametools/animate.c
@@ -188,10 +188,6 @@ int main( int argc, char *argv[] )
     BOOLEAN           highpass = FALSE;
     PassBandParamStruc highpassParam;
 
-    /* test files are version 4 frames */
-    if ( FRAMELIB_VERSION < 4 )
-        return 77;
-
     /* set default values for input */
     dirname = getenv( "LAL_FRAME_PATH" );
 
diff --git a/lalapps/src/frametools/calfacs.c b/lalapps/src/frametools/calfacs.c
index 5d6b8c72f1..e3832a1cff 100644
--- a/lalapps/src/frametools/calfacs.c
+++ b/lalapps/src/frametools/calfacs.c
@@ -131,10 +131,6 @@ int main( int argc, char *argv[] )
     BOOLEAN           highpass = FALSE;
     PassBandParamStruc highpassParam;
 
-    /* test files are version 4 frames */
-    if ( FRAMELIB_VERSION < 4 )
-        return 77;
-
     /* set default values for input */
     dirname = getenv( "LAL_FRAME_PATH" );
 
-- 
2.24.0

